File: Day-25/NCR.js
function factorial(n) {
  let fact = 1;
  for (let i = 1; i <= n; i++) {
    fact = fact * i;
  }

  return fact;
}

function ncr(n, r) {
  const Numerator = factorial(n);
  const Denominator = factorial(r) * factorial(n - r);
  //   console.log(Numerator);
  return Numerator / Denominator;
}

console.log(ncr(8, 2));

File: Day-25/NCR.py
def Factorial(n):
    fact = 1
    for i in range(1, n+1):
        fact *= i
    return fact

def NCR(n, r):
    Numerator = Factorial(n)
    Denomiantor = Factorial(r) * Factorial(n -r)

    return Numerator//Denomiantor

n = int(input())
r = int(input())

print(NCR(n, r))